README.md

```diff
@@ -89,22 +97,18 @@ Modifiy the `webpack.mix.js` file like so:
 ```diff
 mix.js('resources/js/app.js', 'public/js')
     .sass('resources/sass/app.scss', 'public/css')
--   .svelte();    
+-   .svelte();
 +   .svelte({
 +       customElement: true,
-+       tag: null
++       tag: null // to get rid of "no custom element tag name" warning because we are defining components tag name it in app.js file. otherwise you would have to put "<svelte:options tag={null} />" in all of your custom elements.
```

Comment:
i don't know.. this doesn't feel right to me. I think `tag = null` should not be exposed to users because its irrelevant. it should be wrapped within the package itself.

The process of enabling custom element should be as simple as:
```js
.svelte({
     customElement: true
});
```
or
```js
.svelte().withCustomElement();
```
lets see what @morpheus7CS has to say about it.